esp8266-fastled-webserver/include/GradientPalettes.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
SIMPLE FILE REVIEW

This header corresponds to GradientPalettes.cpp, declaring the palettes so that any code in the depot can use them.